README.md

```diff
@@ -1,23 +1,29 @@
-jsonld-signatures
-=================
+# JSON-LD Signatures _(jsonld-signatures)_
 
-[![Build status](https://img.shields.io/github/workflow/status/digitalbazaar/jsonld-signatures/Node.js%20CI)](https://github.com/digitalbazaar/jsonld-signatures/actions?query=workflow%3A%22Node.js+CI%22)
-[![Coverage status](https://img.shields.io/codecov/c/github/digitalbazaar/jsonld-signatures)](https://codecov.io/gh/digitalbazaar/jsonld-signatures)
-[![Dependency Status](https://img.shields.io/david/digitalbazaar/jsonld-signatures.svg)](https://david-dm.org/digitalbazaar/jsonld-signatures)
+[![Node.js CI](https://github.com/digitalbazaar/jsonld-signatures/workflows/Node.js%20CI/badge.svg)](https://github.com/digitalbazaar/jsonld-signatures/actions?query=workflow%3A%22Node.js+CI%22)
+[![NPM Version](https://img.shields.io/npm/v/jsonld-signatures.svg?style=flat-square)](https://npm.im/jsonld-signatures)
```

Comment:
- Why drop the other badges?
- Seems nice to be consistent with the shields.io badges.
- Avoid the special style param unless it's used everywhere.
- I have no idea what the workflow text should be.  These all seem wrong vs just calling it "tests" or similar.  The "Build status" just matched the shields.io badge.
```suggestion
[![Build status](https://img.shields.io/github/workflow/status/digitalbazaar/jsonld-signatures/Node.js%20CI)](https://github.com/digitalbazaar/jsonld-signatures/actions?query=workflow%3A%22Node.js+CI%22)
[![Coverage status](https://img.shields.io/codecov/c/github/digitalbazaar/jsonld-signatures)](https://codecov.io/gh/digitalbazaar/jsonld-signatures)
[![Dependency Status](https://img.shields.io/david/digitalbazaar/jsonld-signatures.svg)](https://david-dm.org/digitalbazaar/jsonld-signatures)
[![NPM Version](https://img.shields.io/npm/v/jsonld-signatures.svg)](https://npm.im/jsonld-signatures)
```